client/lib/cart-values/cart-items.ts

```diff
@@ -399,26 +400,55 @@ export function googleAppsExtraLicenses( properties: {
 }
 
 /**
- * Creates a new shopping cart item for Titan Mail Monthly.
+ * Creates a new shopping cart item for Titan Mail.
  */
-export function titanMailMonthly( properties: {
-	meta?: string;
-	domain?: string;
-	source?: string;
-	quantity?: number | null;
-	extra?: RequestCartProductExtra;
-} ): IncompleteRequestCartProduct {
+function titanMailProduct(
+	properties: {
+		meta?: string;
+		domain?: string;
+		source?: string;
+		quantity?: number | null;
+		extra?: RequestCartProductExtra;
+	},
+	productSlug: string
+) {
 	const domainName = properties.meta ?? properties.domain;
 	if ( ! domainName ) {
 		throw new Error( 'Titan mail requires a domain' );
 	}
 	return {
-		...domainItem( TITAN_MAIL_MONTHLY_SLUG, domainName, properties.source ),
+		...domainItem( productSlug, domainName, properties.source ),
 		quantity: properties.quantity,
 		extra: properties.extra,
 	};
 }
 
+/**
+ * Creates a new shopping cart item for Titan Mail Annually.
+ */
+export function titanMailAnnually( properties: {
+	meta?: string;
+	domain?: string;
+	source?: string;
+	quantity?: number | null;
+	extra?: RequestCartProductExtra;
+} ) {
+	return titanMailProduct( properties, TITAN_MAIL_ANNUALLY_SLUG );
+}
+
+/**
+ * Creates a new shopping cart item for Titan Mail Monthly.
+ */
+export function titanMailMonthly( properties: {
+	meta?: string;
+	domain?: string;
+	source?: string;
+	quantity?: number | null;
+	extra?: RequestCartProductExtra;
+} ): IncompleteRequestCartProduct {
```

Comment:
It looks like we could define an interface to avoid repeating those type definitions:

```suggestion
interface TitanProductProps {
	meta?: string;
	domain?: string;
	source?: string;
	quantity?: number | null;
	extra?: RequestCartProductExtra;
}

export function titanMailMonthly( properties: TitanProductProps ): IncompleteRequestCartProduct {
```